src/composables/useAPI/useAPI.ts

```diff
@@ -0,0 +1,90 @@
+import useNotificationSystem from '@/composables/useNotificationSystem/useNotificationSystem';
+
+/**
+ * API usage composable
+ */
+export default function useAPI() {
+    const baseUrl: string = 'https://api.example.com'; // Replace with your API base URL
+    const bearerToken: string = 'YOUR_BEARER_TOKEN'; // Replace with your Bearer token
+
+    /**
+     * Perform a GET request to the specified API endpoint.
+     * @param {string} endpoint - The API endpoint to fetch data from.
+     * @returns {Promise<ApiResponse>} A Promise that resolves to the API response.
+     * @throws {Error} If the request fails or the response is not OK.
+     */
+    const get = async (endpoint: string): Promise<ApiResponse> => {
```

Comment:
nice2have implementation but actually we don't need GET in VIGAD at all for now

Reply:
If for now should be leave it in or not?